04pointer2/main.go

```diff
@@ -0,0 +1,33 @@
+package main 
+import "fmt"
+
+
+func main(){
+ type name struct{
+        
+       fname string
+       age int
+         }
+    
+    myguy:=name{
+        fname:"Nameless",
+        age: 25,
+    }
+
+
+// value before 
+
+fmt.Printf("FName Before :%s\n",myguy.fname)
+//value after
```

Comment:
If you going to have comments, then let there be a space above the `//value after`